### PR TITLE
JsAssignment6

### PR DESCRIPTION
**Pull request by priyaughade**

### CODE REVIEW

File: priya/javascript/JsAssign6Q1.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (MKadeer17) [inline priya/javascript/JsAssign6Q1.html]: Indentation Little bit
```suggestion
    <head>
```

File: priya/javascript/JsAssign6Q1.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Array</title>
+</head>
+<body>
```
Comment (MKadeer17) [inline priya/javascript/JsAssign6Q1.html]: Same here 4 space before body tag
```suggestion
    <body>
```

File: priya/javascript/JsAssign6Q2.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (MKadeer17) [inline priya/javascript/JsAssign6Q2.html]: Same here in head and body tag

File: priya/javascript/JsAssign6Q3.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (MKadeer17) [inline priya/javascript/JsAssign6Q3.html]: Same here, every page indentation

**Review by MKadeer17 — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do check the files, u have pushed 98 files, push only vch is for the js assignment6 remove extra files.

File: priya/Bootstrap/Assignment1/Bootstrap1.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (saifsyed566) [inline priya/Bootstrap/Assignment1/Bootstrap1.html:3]: head tag indentation is missing

File: priya/Bootstrap/Assignment1/Bootstrap1.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            left: 400px;
+        }
+        .div7{
+            width: 150px; 
+            height: 100px;
+            text-align: start;
+        }
+    </style>
+</head>
+<body>
```
Comment (saifsyed566) [inline priya/Bootstrap/Assignment1/Bootstrap1.html:57]: body tag indentation

File: priya/Bootstrap/Assignment1/Bootstrap1.html [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    <div class="container border-rounded  div1">
+        <div class="jumbotron  bg-success div2">MI-10-MEAN</div>
+        <div class="container border div3 ">Announcements</div>
+            <div class="container border div4">Assignment2-Bootsrap</div>
+            <div class="container border div5" >Assignment1-bootstrap</div>
+            <div class="container border div6">Assignment10-css</div>
+        <div class="jumbotron div7">No work due soon!</div>
+        </div>
```
Comment (saifsyed566) [inline priya/Bootstrap/Assignment1/Bootstrap1.html:65]: <div>tag indentation is incorrect
```suggestion
</div>
```

**Review by saifsyed566 — CHANGES_REQUESTED** [review]

File: priya/javascript/Assignment6/JsAssign6Q1.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+        <p id="result">
+            <p id="demo"></p>
+            <p id="demo1"></p>
+            <p id="demo2"></p>
+        </p>
+        <br>
+        <br>
+        <script language="javascript">
+        function MyArray(){
+            var b,a,result;
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q1.html]: change variable names  in all files like input 1 and input 2 it acn not be a or b.

File: priya/javascript/Assignment6/JsAssign6Q1.html [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+            }
+            document.write("<br>")
+            document.write("<br>")
+             //for name1 name2 
+             for (var i=0 ; i<=b.length-1; i++ ){
+                 document.write( " name" + i + ":" + b[i] + "<br>");
+                 document.write("<br>") ;     
+             }
+        }
+     </script>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q1.html]: there should be four tab space in between parent and child tags so correct the indentation.

File: priya/javascript/Assignment6/JsAssign6Q1.html [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    </body>
+    </html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q1.html]: and new end line and do correct the indentation.

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 11]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">\
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q2.html]: remove extra operator after closing bracket of html

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Array</title>
+    </head>
+    <body>
+        <h4>Enter 5 names here</h4>
+        Name1: <input type="text" id="name1" placeholder="Enter first name " />
+        <br>
+        <br>
+        Name2: <input type="text" id="name2" placeholder="Enter first name " />
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q2.html]: if its anme 2 then change the placeholder accordingly.

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+        <br>
+        <br>
+        Name3: <input type="text" id="name3" placeholder="Enter first name " />
+        <br>
+        <br>
+        Name4: <input type="text" id="name4" placeholder="Enter first name " />
+        <br>
+        <br>
+        Name5: <input type="text" id="name5" placeholder="Enter first name " />
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q2.html]: change the placeholder like enter fifth name

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+            var b, a, c, d, e, cancel, joined, arr;
+            cancel = null;
+            a = document.getElementById("name1").value;
+            b = document.getElementById("name2").value;
+            c = document.getElementById("name3").value;
+            d = document.getElementById("name4").value;
+            e = document.getElementById("name5").value;
+            cancel = document.getElementById("null").value;
+            arr = [a, b, c, d, e, cancel];
+            document.write("<b> The Input Names stored in array : <b> ");
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q2.html]: there  should be no space before colon add space after colon only.

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+                document.write(arr + "<br>");
+                document.write("<br>");
+                //joined with ~
+            document.write("<b> The Input Names joined with ~ :<b> ");
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q2.html]: add space after colon and remove extra space

File: priya/javascript/Assignment6/JsAssign6Q3.html [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Array</title>
+    </head>
+    <body>
+        <h4>Enter 5 Numbers here</h4>
+        Number 1: <input type="text" id="num1" placeholder="Enter first Number "> <br>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q3.html]: remove extra space in all numbers field before " this.

File: priya/javascript/Assignment6/JsAssign6Q3.html [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+        <br>
+        <button onclick="MyArray2()">Enter</button><br>
+        <br>
+        <button id="null">cancel</button>
+        <br>
+        <br>
+        <br>
+        <script language="javascript">
+           function MyArray2(){
+                var b,a,c,d,e,cancel,joined, arr;
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q3.html]: change variable names and there should be always a space after a comma so add it and make a change of variable

File: priya/javascript/Assignment6/JsAssign6Q3.html [hunk 18]
```diff
@@ -0,0 +1,52 @@
...
+                document.write(arr.join(" +"))
+                document.write("<br>");
+                document.write("<br>");
+                document.write("The Addition of input numers are : ");
+                var add = a+ b+ c+ d+ e;
+                document.write(add)
+        }    
+        </script>
+        </body>
+    </html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q3.html]: add new end line

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Array</title>
+    </head>
+    <body>
+        <h4>Enter 5 numbers here</h4>
+        Number 1: <input type="text" id="num1" placeholder="Enter first number "> <br>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: remove extra space in all numbers filed

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+        Number 5: <input type="text" id="num5" placeholder="Enter fifth number "> <br>
+        <br>
+        <button onclick="MyArray2()">Enter</button><br>
+        <br>
+        <button id="null">cancel</button>
+        <br>
+        <br>
+        <script language="javascript">
+            function MyArray2(){
+                var b,a,c,d,e,fact, arr;
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: change the variable names and add a space after comma

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+                cancel = null;
+                //for input names
+                a= parseInt(document.getElementById("num1").value);
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: there should be always a space before and after operator so make this everywhere except in html tags

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 22]
```diff
@@ -0,0 +1,57 @@
...
+                b=parseInt(document.getElementById("num2").value);
+                c=parseInt(document.getElementById("num3").value);
+                d=parseInt(document.getElementById("num4").value);
+                e=parseInt(document.getElementById("num5").value);
+                cancel=document.getElementById("null").value;
+                arr=[a,b,c,d,e]
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: add space after comma

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 23]
```diff
@@ -0,0 +1,57 @@
...
+                document.write(arr);
+                document.write("<br>");
+                document.write("<br>");
+                var sum=0;
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: add space after and before all operator in all files

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+                        }
+                        sum = sum + fact;   
+                        document.write("<br>");
+                    }
+                    document.write("The sum of all factorials are : " +sum)
+                    document.write("<br>");
+                }      
+                </script>
+            </body>
+    </html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: add new end line

File: priya/javascript/Assignment6/JsAssign6Q5.html [hunk 25]
```diff
@@ -0,0 +1,65 @@
...
+                      document.write(fact)
+                      document.write("<br>");
+                      sum.push(fact);   
+                    }
+                    document.write("New array of factorial : " + sum )
+                    document.write("<br>");
+                    sum1=sum.sort(function(a,b){return a-b});
+                    document.write(" The array of factorial by descending order : ")
+                    document.write(sum1);
+                    // for (var i=0 ; i<=sum1.length-1; i++ ){
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q5.html]: do not push commented code

File: priya/javascript/Assignment6/JsAssign6Q5.html [hunk 26]
```diff
@@ -0,0 +1,65 @@
...
+                      //document.write( sum1[i] + ",") ;
+                      //console.log("<br>") 
+                      // }
+                    }
+              </script>
+        </body>
+  </html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q5.html]: add new end line in all files

File: priya/javascript/JsAssign6Q1.html (hunk 1 above)
Comment (SnehaMakade) [inline priya/javascript/JsAssign6Q1.html]: make a proper indentation

File: priya/javascript/JsAssign6Q1.html [hunk 27]
```diff
@@ -0,0 +1,59 @@
...
+    <button onclick="MyArray()">Enter</button>
+    <p id="result">
+        <p id="demo"></p>
+        <p id="demo1"></p>
+        <p id="demo2"></p>
+    </p>
+    <br>
+    <br>
+    <script language="javascript">
+
```
Comment (SnehaMakade) [inline priya/javascript/JsAssign6Q1.html]: remove extra line and space also

File: priya/javascript/JsAssign6Q1.html [hunk 28]
```diff
@@ -0,0 +1,59 @@
...
+              document.write("<br>")
+              document.write("<br>")
+
+              //for name1 name2 
+              for (var i=0 ; i<=b.length-1; i++ ){
+                  document.write( " name" + i + ":" + b[i] + "<br>");
+                  document.write("<br>") ;     
+              }
+            }
+
```
Comment (SnehaMakade) [inline priya/javascript/JsAssign6Q1.html]: remove extra lines

File: priya/javascript/JsAssign6Q2.html (hunk 3 above)
Comment (SnehaMakade) [inline priya/javascript/JsAssign6Q2.html]: indentation should be proper in all files

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes in all files, changes of variable names removing of extra space and lines and do proper indentation in all files

File: priya/javascript/Assignment6/JsAssign6Q1.html [hunk 29]
```diff
@@ -8,42 +8,37 @@
...
-                 document.write( " name" + i + ":" + b[i] + "<br>");
-                 document.write("<br>") ;     
-             }
-        }
-     </script>
+        </script>
     </body>
-    </html>
\ No newline at end of file
+</html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q1.html]: add end line in all files

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 30]
```diff
@@ -8,64 +8,43 @@
...
-               upper1 = upper2.reverse();
-               upper1 = upper1.map(function (x) {
-                   return x.toUpperCase();
-                });
-                document.write(upper1);
-            }
-            </script>
-        </body>
-    </html>
+                document.write("Input Names with Assending order  : <br>");
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q2.html:32]: there should be no space befor colon remove the space before colon

File: priya/javascript/Assignment6/JsAssign6Q3.html [hunk 31]
```diff
@@ -8,45 +8,29 @@
...
-                a= parseInt(document.getElementById("num1").value);
-                b=parseInt(document.getElementById("num2").value);
-                c=parseInt(document.getElementById("num3").value);
-                d=parseInt(document.getElementById("num4").value);
-                e=parseInt(document.getElementById("num5").value);
-                cancel=document.getElementById("null").value;
-                arr=[a,b,c,d,e]
+                var sum,arr;
+                arr = [];
+                for (var i=0; i<=4; i++){
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q3.html:18]: add a space before and after operator in all files

File: priya/javascript/Assignment6/JsAssign6Q3.html [hunk 32]
```diff
@@ -8,45 +8,29 @@
...
+                    sum= sum+arr[i];
+                }
+                document.write("The Addition of input numers are : " + sum);
+             }    
         </script>
-        </body>
-    </html>
\ No newline at end of file
+     </body>
+</html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q3.html]: add end line

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 33]
```diff
@@ -8,50 +8,34 @@
...
-                    document.write("<br>");
-                }      
-                </script>
-            </body>
-    </html>
\ No newline at end of file
+                    }      
+        </script>
+    </body>
+</html>
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q4.html]: add end line in all files and add a space after and before each operator

File: priya/javascript/Assignment6/JsAssign6Q5.html [hunk 34]
```diff
@@ -8,54 +8,39 @@
...
-              var b,a,c,d,e,fact, arr;
-              cancel = null;
-                a= parseInt(document.getElementById("num1").value);
-                b=parseInt(document.getElementById("num2").value);
-                c=parseInt(document.getElementById("num3").value);
-                d=parseInt(document.getElementById("num4").value);
-                e=parseInt(document.getElementById("num5").value);
-                cancel=document.getElementById("null").value;
-                arr=[a,b,c,d,e]
+            var sum,arr,fact;
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q5.html:17]: add a space after comma and colon in each file

File: priya/javascript/Assignment6/JsAssign6Q5.html [hunk 35]
```diff
@@ -8,54 +8,39 @@
...
+                          console.log(fact);
+                          document.write(fact)
+                              document.write("<br>");
+                              sum.push(fact);   
+                            }
+                            document.write("New array of factorial : " + sum )
+                            document.write("<br>");
+                            sum1=sum.sort(function(a,b){return a-b});
+                              document.write(" The array of factorial by descending order : ")
+                                document.write(sum1);
```
Comment (SnehaMakade) [inline priya/javascript/Assignment6/JsAssign6Q5.html:42]: dont push commented code on git.remove it

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes

File: priya/javascript/Assignment6/JsAssign6Q1.html [hunk 36]
```diff
@@ -41,4 +41,4 @@ <h4>Enter 5 names here</h4>
             }
         </script>
     </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (Snehal-113) [inline priya/javascript/Assignment6/JsAssign6Q1.html:36]: Not getting the expected output.Either take 5 text fields or 5 times prompt and then display the output

Comment (Snehal-113) [reply priya/javascript/Assignment6/JsAssign6Q1.html:36]: indentation is not done properly.use prettier

File: priya/javascript/Assignment6/JsAssign6Q2.html [hunk 37]
```diff
@@ -48,3 +48,4 @@ <h4>Enter 5 names here</h4>
         </script>
     </body>
 </html>
```
Comment (Snehal-113) [inline priya/javascript/Assignment6/JsAssign6Q2.html:50]: need to accept names till the user presses cancel in prompt dialog box and not only 5

File: priya/javascript/Assignment6/JsAssign6Q4.html [hunk 38]
```diff
@@ -38,4 +38,5 @@ <h4>Enter 5 numbers here</h4>
                     }      
         </script>
     </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (Snehal-113) [inline priya/javascript/Assignment6/JsAssign6Q4.html:41]: same change as mentioned in teh above program

File: priya/javascript/Assignment6/JsAssign6Q5.html [hunk 39]
```diff
@@ -47,4 +47,5 @@ <h4>Enter 5 number here</h4>
                     }
               </script>
         </body>
-  </html>
\ No newline at end of file
+</html>
```
Comment (Snehal-113) [inline priya/javascript/Assignment6/JsAssign6Q5.html:50]: same change

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Review by Snehal-113 — APPROVED** [review]

**Review by saifsyed566 — APPROVED** [review]

**Pull request merged**